docs/website/mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: IREE
-site_url: https://openxla.github.io/iree/
+site_url: https://www.iree.dev/
```

Comment:
Hmm... looks like the 404 page is broken with the current URL. Updating this site url may help.

https://www.iree.dev/404
```
 GET https://www.iree.dev/iree/assets/stylesheets/main.0e669242.min.css net::ERR_ABORTED 404 (Not Found)
 GET https://www.iree.dev/iree/assets/stylesheets/palette.85d0ee34.min.css net::ERR_ABORTED 404 (Not Found)
 GET https://www.iree.dev/iree/assets/stylesheets/extra.css net::ERR_ABORTED 404 (Not Found)
 GET https://www.iree.dev/iree/assets/stylesheets/openxla.css net::ERR_ABORTED 404 (Not Found)
 GET https://www.iree.dev/iree/assets/javascripts/bundle.78eede0e.min.js net::ERR_ABORTED 404 (Not Found)
 GET https://www.iree.dev/iree/assets/images/openxla-logo-lockup-white.svg 404 (Not Found)
```
Note the extra `iree/` in those paths. The URLs in the .html are of the form `<link rel="stylesheet" href="/iree/assets/stylesheets/extra.css">`

Reply:
Nice, this fixed the 404 page (e.g. https://iree.dev/404)